SingleSource/UnitTests/Atomic/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Link the Clang built libatomic.
+execute_process(COMMAND ${CMAKE_C_COMPILER} --print-file-name=libclang_rt.atomic.so
```

Comment:
That library isn't built by default, so almost nobody will have a copy. I assume we need to have some conditionals here to skip testing if the library isn't present? 

Potentially we should additionally integration-test against `-latomic` (even if that comes via GCC), since that's what real users actually specify.

Reply:
Done. I'm not great with CMake -- what are better ways to do what I have here?